lib/region/europe/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
```

Comment:
There was some discussion between @aterranova-bv and I around this that's getting hidden since I renamed this file (`EU` - > `europe`). Long story short, objects lookups are a very fast O(1), Array lookups are a slow O(n). This is because, in most javascript engines, Objects are hash maps while "Arrays" are actually linked lists. [Here's a performance test to show the difference](http://jsperf.com/jst-7/2).

Javascript Arrays certainly have their place. If you plan on modifying your list or iterating over it, Arrays are ideal. However, for constant lists that remain unchanged and are used only for lookup operations, Objects are king. Performance isn't really an issue for this module, but I still made it an object for common practice's sake.